concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -193,8 +193,8 @@ def test_provider_forces_subtask_results_but_it_was_sent_to_too_late_or_too_soon
                 timestamp       = "2018-03-05 10:00:15",
                 subtask_id      = "2",
                 task_to_compute = self._get_deserialized_task_to_compute(
-                    timestamp   = "2018-02-05 10:00:00",
-                    deadline    = "2018-02-05 10:00:10",
+                    timestamp   = "2018-03-05 10:00:00",
```

Comment:
Why you had to change day ?

Reply:
All others timestamps and freeze_time where `2018-03-05`. With new time window checks we use `task_to_compute.deadline` in comparison to `current_time`. I needed to change one or the other so they could match.

Reply:
:+1: